buildflow/io/clickhouse/providers/clickhouse_providers.py

```diff
@@ -0,0 +1,27 @@
+from buildflow.core.credentials import ClickhouseCredentials
```

Comment:
we actually just updated our API a bit so you don't have to implement the provider's anymore. Instead a "primitive" has the sink method directly know. You should just be able to delete this, update the methods on your primitive to return your sink directly.

Here's the work in progress docs for the "new" way: https://docs.buildflow.dev/buildflow/programming-guide/primitives#creating-custom-primitives